### ISSUE TITLE
Opt out of CSS framework

### ISSUE THREAD

**Issue by claytoncrockett** [wdjungst/create-repack-app#31]
Been looking into creating a flag to opt out of using a css framework.  This would require basically creating an alternate client setup consisting of components that don't use semantic for their styling.  Wondering if anyone else has interest in this use case, if there is interest I would be happy to start poking through it.

**Comment by wdjungst** [comment]
As an alternative you could use styled components with the same name and import them from a file  and simply exclude the css file.  This way you would have to change the components except for the import path.

**Comment by claytoncrockett** [comment]
Yeah that works for me.  Correct me if I'm wrong, but this would require still making duplicates of all of the components, where the only different is the
`import { Form } from "semantic-ui-react";`
would be changed to like
`import { Form } from "custom-styled-components";`

(generic example)

Or is there a way to conditionally insert that import and keep it to just the original files?
